### PR TITLE
Removed re: message_id

### PR DESCRIPTION
**Pull request by Joshix-1**
I deleted the old branch with the pr by mistake

### CODE REVIEW

**Comment by Vampire** [comment]
For the next time, just push the branch again to your repo and ask for a reopen of the PR, now the review history is out of context and hard to follow. ;-)

File: src/main/java/org/javacord/bot/listeners/CommandCleanupListener.java [hunk 1]
```diff
@@ -43,14 +44,18 @@ public void onMessageDelete(MessageDeleteEvent event) {
...
+                .filter(text -> text.startsWith(tracker))
+                .isPresent();
     }
 
-
+    private static String longToBinaryBlankString(long l)  {
+        return Long.toBinaryString(l)
+                .replace('0', '\u200B')
+                .replace('1', '\u200D');
+    }
```
Comment (Vampire) [inline src/main/java/org/javacord/bot/listeners/CommandCleanupListener.java:60]: What do you think about terminating the binary blank string to not later match on prefixes as you changed to `startsWith`?
Maybe another zero-width characer. Maybe `\u200C`?

Comment (Vampire) [reply src/main/java/org/javacord/bot/listeners/CommandCleanupListener.java:60]: Or maybe then B for 0, C for 1 and D as terminator

Comment (Vampire) [reply src/main/java/org/javacord/bot/listeners/CommandCleanupListener.java:61]: Any reason to not have B, C, D in order? :-)

Comment (Joshix-1) [reply src/main/java/org/javacord/bot/listeners/CommandCleanupListener.java:61]: I just added the part with the C :-)

**Comment by Vampire** [comment]
Sorry for the delay, I'd say it looks good now.
Would you mind squashing your commits into one, then I think we are good to go. :-)

**Comment by Vampire** [comment]
Btw. you should give a note when you are ready with incorporating review feedback, either by dropping a note, or by requesting rereview with the GitHub function: 
grafik

Not for every event, like a force-push there is a notification.
I just happened to look again manually. ;-)

**Comment by Vampire** [comment]
And one last nit-pick about commit messages,
https://chris.beams.io/posts/git-commit/ is a great read and recipe, at least generally.
I don't follow it to the word or require it personally, like the 50 chars limit, the 72 chars line break and always having a body (but still separating with a newline if present).
But most things written there regarding mood of the subject, formatting and so on are a very good idea to follow. ;-)
